src/components/options/OptionsContainer.js

```diff
@@ -29,6 +28,58 @@ class OptionsContainer extends Component {
     };
   }
 
+  constructor(props) {
+    super(props);
+
+    this.handleChangeDelay = this.handleChangeDelay.bind(this);
+    this.handleClickMinus = this.handleClickMinus.bind(this);
+    this.handleClickPlus = this.handleClickPlus.bind(this);
+    this.confirmChangeDelay = this.confirmChangeDelay.bind(this);
+
+    this.confirmDelayTimeout = null;
+
+    this.state = {
+      delay: props.options.get('timeToClose') ? props.options.get('timeToClose') : 30,
```

Comment:
Weird to show 30 for someone who does not have set this option, and so does not have the disconnect enabled.
cf: https://github.com/secretin/secretin-app/pull/22/files#diff-57aa11691094c3295e1f9dcbe8432ba2R61